supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -143,15 +179,15 @@ To italicize a user-replaced value in a code block, you must add an attribute to
 
   [subs="+quotes"]
   ----
-  $ oc describe node __<node_name>__
+  $ *oc describe node __<node_name>__*
```

Comment:
The division from the lead-in sentence is marked by the code block's shaded box, so I'm not really sure bolding the command is necessary. I understand bolding a command to distinguish it from command output that resides in the same code block.

Reply:
Well, the original pitch for this change partly stemmed from the fact that the light-grey shading of the code block in the current stylesheets does not really do much to highlight whatever it contains. 

In addition, for the sake of consistency, we agreed that commands in code blocks should be bolded regardless of whether they are accompanied by the output or not.

Reply:
Thanks for the reply. Phew, a lot of code blocks will need to be updates after this update!  Would detailing the "quotes" subs attribute be important so that a writer knows the importance of the attribute on how the AsciiDoc processor renders asterisk symbol as bold highlighting?

A link could be provided that points to https://docs.asciidoctor.org/asciidoc/latest/subs/apply-subs-to-blocks/ . I see a few links to Asciidoctor docs in the RH SSG.